main/src/main/java/cgeo/geocaching/unifiedmap/layers/PositionLayer.java

```diff
@@ -50,6 +53,16 @@ public PositionLayer(final AppCompatActivity activity, final GeoItemLayer<String
                                     .setBitmap(markerPosition).build())
                             .buildUpon().setZLevel(LayerHelper.ZINDEX_POSITION).build());
 
+            if (locationWrapper.location.hasAltitude() && Settings.showElevation()) {
+                layer.put(KEY_ELEVATION,
+                        GeoPrimitive.createMarker(new Geopoint(locationWrapper.location), GeoIcon.builder()
+                                        .setRotation(0f)
+                                        .setFlat(true)
+                                        .setBitmap(MapUtils.getElevationBitmap(activity.getResources(), markerPosition.getHeight(), locationWrapper.location.getAltitude()))
```

Comment:
Maybe use `GeoIcon.Builder.setText` instead? (Or use `TextBitmapProvider` directly to have more control)

Reply:
thanks for the two pointers, will look into it

Reply:
Ok, I got some results when using `GeoIcon.builder().setBitmap(new GeoIcon.TextBitmapProvider(...)`, but that leaves two questions open:
- Currently the elevation info gets centered beneath the position marker. How can I position the text "south" of the elevation marker? (In my bitmap variant I create a blank bitmap part the size of the marker as a spacer.)
- How can I create round edges for the background around the elevation info?

|current|instead of|
|---|---|
|![image](https://github.com/cgeo/cgeo/assets/3754370/f1a475df-a5b9-479a-8f1e-e13ebae18b17) | ![image](https://github.com/cgeo/cgeo/assets/3754370/09124f86-0ad6-4449-afa0-3d60b2fdb967)|

Reply:
For position I believe there's a method setYShift() or something like this on the builder

Round corners are not supported currently I am afraid. This must be added to the ImageUtils method and then also to the Text bitmap provider first.

If this is too complex, maybe leave I
PR as was before and I can refactor some other time. It's no biggie

Reply:
Update: the method I was looking for above is called `setYAnchor()`, but it is currently not suited for your use case (it is only implemented to move the image anchor within the image bounds), So also here adaption would be needed. As you already suggested: best merge this PR as is onto release, we can refactor later (on master).